main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -100,7 +100,7 @@ DataTableView.prototype.render = function() {
     '</div>' +
     '<div bind="dataTableContainer" class="data-table-container">' +
       '<table class="data-table">'+
-        '<thead bind="tableHeader" class="data-table-header">'+
+        '<thead bind="tableHeader" class="data-table-header" style="background: #ebeef8">'+
```

Comment:
Shouldn't this style be applied from the CSS instead?

Reply:
I tried, but I dont know applying from Styles was making the complete table background Blue ? It was quite weird .

Reply:
If you can investigate again it would be great!

Reply:
@wetneb Fixed it! The issue was I was applying styling at the wrong class which was letting the header as well as body to be colored in background.